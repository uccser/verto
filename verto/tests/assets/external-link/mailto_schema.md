Check out this [email](mailto:bob@example.com).
