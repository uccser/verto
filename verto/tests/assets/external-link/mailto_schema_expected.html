<p>Check out this <a href="mailto:bob@example.com" target="_blank">email</a>.</p>
